lib/PuppeteerSharp/Browser.cs

```diff
@@ -82,6 +89,11 @@ internal void ChangeTarget(TargetInfo targetInfo)
             return version.product.ToString();
         }
 
+        public void Disconnect()
```

Comment:
Could you add an Xml doc here?
Just copy these cool guys https://github.com/GoogleChrome/puppeteer/blob/v1.0.0/docs/api.md